openmdao/components/implicit_func_comp.py

```diff
@@ -0,0 +1,191 @@
+"""Define the ImplicitFuncComp class."""
+
+
+import numpy as np
+from numpy import asarray, isscalar
+from itertools import chain
```

Comment:
itertools import should be above numpy for better PEP.

Reply:
fixed